eng/build.sh

```diff
@@ -131,6 +138,14 @@ while [[ $# > 0 ]]; do
   esac
 done
 
+if [[ "$buildtests" == true ]]; then
```

Comment:
The condition is wrong because you're comparing a string against a boolean. "true" == true... I guess that's why the build is failing to find test work items.

Reply:
the condition is correct: https://github.com/dotnet/corefx/blob/master/eng/common/tools.sh#L35

I'm still searching for the actual issue here...

Reply:
Ah right. Didn't know that about shell. Good to know... I see now that you fixed the issue, stupid dollar sign 😆

Reply:
I was literally looking at it for 10min straight and couldn't see the error.

Reply:
Same here. I suspected it was an error like that so I was trying to help you spot it and completely missed it.

Reply:
thanks, I highly appreciate that